data/data/agent/systemd/units/create-cluster-and-infraenv.service.template

```diff
@@ -13,7 +13,7 @@ Environment=OPENSHIFT_INSTALL_RELEASE_IMAGE_MIRROR={{.ReleaseImageMirror}}
 EnvironmentFile=/usr/local/share/assisted-service/agent-images.env
 ExecStartPre=/bin/rm -f %t/%n.ctr-id
 ExecStartPre=/usr/local/bin/wait-for-assisted-service.sh
-ExecStart=podman run --cidfile=%t/%n.ctr-id --cgroups=no-conmon --log-driver=journald --rm --pod-id-file=%t/assisted-service-pod.pod-id --replace --name=create-cluster-and-infraenv -v /etc/assisted/manifests:/manifests -v /etc/assisted/extra-manifests:/extra-manifests -v /etc/pki/ca-trust:/etc/pki/ca-trust:z --env SERVICE_BASE_URL --env OPENSHIFT_INSTALL_RELEASE_IMAGE_MIRROR $SERVICE_IMAGE /usr/local/bin/agent-installer-client register
+ExecStart=podman run --net host --cidfile=%t/%n.ctr-id --cgroups=no-conmon --log-driver=journald --rm --pod-id-file=%t/assisted-service-pod.pod-id --replace --name=create-cluster-and-infraenv -v /etc/assisted/manifests:/manifests -v /etc/assisted/extra-manifests:/extra-manifests -v /etc/pki/ca-trust:/etc/pki/ca-trust:z --env SERVICE_BASE_URL --env OPENSHIFT_INSTALL_RELEASE_IMAGE_MIRROR $SERVICE_IMAGE /usr/local/bin/agent-installer-client register
```

Comment:
Do containers inside the pod need `--net=host` when the pod already has `--net=host`?

Reply:
I've tried this (adding `--net host` to assisted-service-pod.service and removing it from the containers) but it didn't work. It looks like the IPv6 address was not exposed as I get:
Sep 06 15:25:30 master-0.ostest.test.metalkube.org start-agent.sh[3644]: curl: (7) Failed to connect to fd2e:6f44:5dd8:c956::50 port 8090: Connection refused

Reply:
then i guess we can merge it as is?